core/txpool/blobpool/blobpool.go

```diff
@@ -334,9 +334,11 @@ func New(config Config, chain BlockChain) *BlobPool {
 	}
 }
 
-func (p *BlobPool) SetAstriaOrdered(types.Transactions) {}
-func (p *BlobPool) ClearAstriaOrdered()                 {}
-func (p *BlobPool) AstriaOrdered() *types.Transactions  { return &types.Transactions{} }
+func (p *BlobPool) SetAstriaOrdered(types.Transactions)              {}
+func (p *BlobPool) ClearAstriaOrdered()                              {}
+func (p *BlobPool) UpdateAstriaExcludedFromBlock(*types.Transaction) {}
```

Comment:
nit on naming... perhaps `NewAstriaOrderedExcludedTx`? naming here is always going to be awkward as it's actually a method on a subtype, its semantically kinda wrong. 

Not for this PR, but the adding stuff to the mempool has always felt a bit awkward. Might makes more sense to add payload argument or something, or even create a new subpool.

Reply:
I m not sure if adding a new subpool would make sense since we kind of need access to the legacy pool to remove txs.